### ISSUE TITLE
Turning on Gradient Clipping

### ISSUE THREAD

**Issue by Steven-Palayew** [amaiya/ktrain#432]
Hello! 

I tried turning on gradient clipping by adding lines four and five to my code here, 
![image](https://user-images.githubusercontent.com/54290594/158466507-0f2acf82-f974-4670-925d-f83e46f09498.png). 

This was based on [this](https://github.com/amaiya/ktrain/blob/master/FAQ.md#how-do-i-use-custom-loss-functions-or-optimizers) FAQ section. 

However, I get a "No gradients provided for any variable" error after calling learner.fit_onecycle. 

I was wondering what my error is here and/or if there's a better way of turning on gradient clipping?

Thank you so much in advance!

**Comment by amaiya** [comment]
Try this:
```python
loss_fn = tf.keras.losses.CategoricalCrossentropy(from_logits=True)
opt = tf.keras.optimizers.Adam(learning_rate=5e-5, clipnorm=1.0)
model.compile(loss=loss_fn,
              optimizer=opt,
              metrics=['accuracy'])
```

**Comment by glane2609** [comment]
The code was running fine for 0.30.0 , now I'm getting following error
![image](https://user-images.githubusercontent.com/62784645/212054383-7c2180ee-7260-4539-a045-22acc06e3421.png)

this is my code
`import ktrain
from ktrain import text
from ktrain.text import texts_from_array
from sklearn.model_selection import train_test_split

classes =  list(set(reviews_df.score))

X_train, X_test, y_train, y_test = train_test_split(reviews_df.content, reviews_df.score, test_size=0.2, random_state=42)

# Pre Processing 
trn, val, preproc = text.texts_from_array(x_train=X_train.values, y_train=y_train.values,
                                          x_test=X_test.values, y_test=y_test.values,
                                                                   class_names=classes,
                                                                   val_pct=0.1, 
                                                                   preprocess_mode='distilbert')


# Model 
model = text.text_classifier('distilbert', train_data=trn, preproc=preproc)`
